src/api/store/media_library.py

```diff
@@ -21,6 +22,16 @@ class MediaLibraryStore:
     def __init__(self):
         self.name = "MediaLibrary Store/DB"
 
+    def read_image(self, args):
+        try:
+            id = args.get("media_id", None)
+            image = Media.objects.filter(pk=id).get()
```

Comment:
I am not sure this query will work. try `Media.objects.filter(pk=id).first()`

Reply:
You do realise I did the test the whole thing before I made the PR right? 🤣 🤣 🤣. 
It works, try it locally yourself.